Remotion/Data/DomainObjects/DomainObject.cs

```diff
@@ -392,9 +390,10 @@ protected void BaseGetObjectData (SerializationInfo info, StreamingContext conte
     /// <exception cref="ArgumentNullException">The <paramref name="id"/> or <paramref name="rootTransaction"/> parameter is null.</exception>
     /// <exception cref="InvalidOperationException">This <see cref="DomainObject"/> has already been initialized.</exception>
     /// <remarks>This method is always called exactly once per <see cref="DomainObject"/> instance by the framework. It sets the object's 
-    /// <see cref="ID"/> and enlists it with the given <see cref="DomainObjects.ClientTransaction"/>.</remarks>
+    /// <see cref="ID"/> and enlists it with the given <see cref="DomainObjects.ClientTransaction"/> as well as the proper <see cref="DomainObjectTransactionContextImplementation"/>.</remarks>
```

Comment:
```suggestion
    /// <see cref="ID"/> and enlists it with the given <see cref="DomainObjects.ClientTransaction"/>.</remarks>
```
That's an implementation detail we don't need to document here.